src/pagerduty/tests/main_test.bal

```diff
@@ -0,0 +1,515 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/time;
+
+Configuration pagerdutyConfig = {
+    oauth2Config: {
+        accessToken: "mL7WHpakjRZshG7qJiey"
```

Comment:
I think you shouldn't add the access token like this. You'll have to pass it as a github secret. https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

Reply:
Yes. Please revoke this token since it is already committed.